src/buffer/out/UTextAdapter.cpp

```diff
@@ -186,23 +186,22 @@ catch (...)
 static UBool U_CALLCONV utextAccess(UText* ut, int64_t nativeIndex, UBool forward) noexcept
 try
 {
-    if (nativeIndex < 0)
-    {
-        nativeIndex = 0;
-    }
-
     auto neededIndex = nativeIndex;
+    // This will make it simpler for us to search the row that contains the nativeIndex,
+    // because we'll now only need to check for `start<=index<limit` and nothing else.

```

Comment:
Reduce your diff NOW by applying the patented whitespace-reduction to your browser: https://github.com/microsoft/terminal/pull/17361/files?w=1
Results may vary.